apps/autoscheduler/train_cost_model.cpp

```diff
@@ -46,6 +48,23 @@ bool ends_with(const string &str, const string &suffix) {
     return true;
 }
 
+void log_best_schedule(int best_id, float best_runtime) {
+    const int batch = best_id / 100;
```

Comment:
The /100 %100 thing is a function of the way in which the autotuning script is currently set up, and probably doesn't belong in the binary that trains the cost model. The filename of the best sample file could be tracked below just as easily as its schedule id, so there's no need to recreate it here. In the comparison of `runtime < best_runtime` you can just grab the variable `s` as well to keep track of the sample file that had the lowest runtime in it.

Reply:
Done